src/image-copy/azext_imagecopy/cli_utils.py

```diff
@@ -86,3 +88,9 @@ def get_storage_account_id_from_blob_path(cmd, blob_path, resource_group, subscr
         namespace='Microsoft.Storage', type='storageAccounts', name=storage_account_name)
 
     return storage_account_id
+
+
+def format_cmd(cmd: List[str]):
+    if len(cmd) > 3:
+        return 'Running: az ' + ' '.join(cmd[3:])
```

Comment:
Is this safe on all platforms with different entry scripts? If so, add some comments, otherwise using the magic number `3` in `cmd[3:]` may cause trouble.

Reply:
You are right. I haven't tested in other platforms except Powershell.